src/main/java/eu/europeana/metis/sandbox/common/amqp/RecordMessageConverter.java

```diff
@@ -36,6 +46,12 @@ public Message toMessage(Object object, MessageProperties messageProperties) {
 
     Event recordEvent = (Event) object;
     Record record = recordEvent.getBody();
+    String recordException;
+    try {
+      recordException = recordEvent.getException() != null? objectMapper.writeValueAsString(recordEvent.getException()) : null;
```

Comment:
Why do we need to do this? `recordEvent.getException()` is already a string. So we don't need to convert this, do we?

Reply:
I think there was a merge issue when I merged developed into my branch, I will fix it and then it will make sense

Reply:
At the end I took another approach, but since this comment does not show anymore in the review I just want to make the statement here before resolving this one